var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -210,16 +210,11 @@ class Dealii(CMakePackage, CudaPackage):
     depends_on('symengine@0.6:', when='@9.2:+symengine')
     depends_on('tbb',            when='+threads')
     # do not require +rol to make concretization of xsdk possible
-    depends_on('trilinos+amesos+aztec+epetra+ifpack+ml+muelu+sacado+teuchos',
-               when='+trilinos+mpi~int64~cuda')
-    depends_on('trilinos+amesos+aztec+epetra+ifpack+ml+muelu+sacado+teuchos~hypre',
-               when='+trilinos+mpi+int64~cuda')
+    depends_on('trilinos+amesos+aztec+epetra+ifpack+ml+muelu+sacado', when='+trilinos')
+    depends_on('trilinos~hypre', when='+trilinos+int64')
     # TODO: temporary disable Tpetra when using CUDA due to
     # namespace "Kokkos::Impl" has no member "cuda_abort"
-    depends_on('trilinos@master+amesos+aztec+epetra+ifpack+ml+muelu+rol+sacado+teuchos~amesos2~ifpack2~intrepid2~kokkos~tpetra~zoltan2',
-               when='+trilinos+mpi~int64+cuda')
-    depends_on('trilinos@master+amesos+aztec+epetra+ifpack+ml+muelu+rol+sacado+teuchos~hypre~amesos2~ifpack2~intrepid2~kokkos~tpetra~zoltan2',
-               when='+trilinos+mpi+int64+cuda')
+    depends_on('trilinos@master+rol~amesos2~ifpack2~intrepid2~kokkos~tpetra~zoltan2', when='+trilinos+cuda')
```

Comment:
Thanks for the ping about these changes @sethrj. It looks much better now with all of these simplifications!